themes/navy/source/css/_partial/index.styl

```diff
@@ -45,6 +45,39 @@
   @media mq-normal
     display: block
 
+#intro-news-list
+  background-color: #10171e
+  margin: -1px 0
+
+#intro-news-flex
+  display: flex
+  flex-flow: column
+  @media mq-normal
+    flex-flow: row wrap
+
+.intro-news-wrap
+  padding: 36px 30px
+  text-decoration: none
+  &:hover
+    background-color: #25a1f0
+  @media mq-normal
+    flex: 0 0 50%
+  @media screen and (min-width: 960px)
+    flex: 0 0 25%
+
+.intro-news-time
+  font-size: 14px
+  color: #888
```

Comment:
I would have preferred banner font color "rgba(255,255,255,.8)", but it maybe too bright.

Reply:
Changed to `rgba(255,255,255,.6)`